src/OrbitGl/App.cpp

```diff
@@ -606,6 +606,15 @@ void OrbitApp::OnErrorEnablingUserSpaceInstrumentationEvent(
   });
 }
 
+void OrbitApp::OnInfoEnablingUserSpaceInstrumentationEvent(
+    orbit_grpc_protos::InfoEnablingUserSpaceInstrumentationEvent info_event) {
+  main_thread_executor_->Schedule([this, info_event = std::move(info_event)]() {
+    main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kSevereWarning,
```

Comment:
The other `kSevereWarning` cases are associated to (dismissable) message boxes, and to "Error" events. So I think that this should only be `kWarning`.